docs/sources/operators-guide/tools/mimirtool.md

```diff
@@ -72,6 +72,9 @@ The following sections outline the commands that you can run against Grafana Mim
 
 The following commands interact with Grafana Mimir Alertmanager configuration and alert template files.
 
+> **Note:** The Mimir Alertmanager exposes the configuration API according to the path set by the `-server.path-prefix` flag. It does not use the path set by the `-http.alertmanager-http-prefix` flag.
+> With the default configuration of `-server.path-prefix`, the Alertmanager URL used as the `mimirtool` `--address` flag has no path portion.
```

Comment:
I would try to clarify it. I don't know if my edit is better, but I would try to make it easier to understand.

```suggestion
> With the default configuration of `-server.path-prefix`, `mimirtool` `--address` flag should be set to an URL with the `/` path.
```

Reply:
I tried thinking of something more clear, but I couldn't do much better.

what do you think about just giving an example?
```suggestion
> With the default configuration of `-server.path-prefix`, the Alertmanager URL used as the `mimirtool` `--address` flag has no path portion: for example `http://alertmanager.mimir.example.com/`.
```

Reply:
The example could help. @osg-grafana could help further :)